electron-settings module missing

After cloning:

```
C:\Users\user\dalton> npm install
[...]
C:\Users\user\dalton> npm start

> Dalton@1.0.0 start C:\Users\user\dalton
> electron .


App threw an error during load
Error: Cannot find module 'electron-settings'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (C:\Users\user\AppData\Roaming\npm\node_modules\electron\dist\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\user\dalton\main.js:12:18)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
```

I tried node main.js as well, but that is even missing electron itself. Any Ideas? Electron is present an on the path:

```
> where electron
C:\Users\user\AppData\Roaming\npm\electron
C:\Users\user\AppData\Roaming\npm\electron.cmd

>electron --version
v1.4.3
```

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
     "electron-prebuilt": "^1.2.0"
   },
   "dependencies": {
+    "electron-settings": "^2.2.1",
     "esprima": "^2.7.2",
     "underscore": "^1.8.3"
   }

